packages/platform/atoms/booker/BookerPlatformWrapper.tsx

```diff
@@ -198,7 +217,7 @@ const BookerPlatformWrapperComponent = (
     rescheduleUid: props.rescheduleUid ?? null,
     bookingUid: props.bookingUid ?? null,
     layout: layout,
-    org: props.entity?.orgSlug,
+    org: resolvedOrgSlug,
```

Comment:
Ah, my bad - i forgot that we already pass `props.entity?.orgSlug` to org. Which means that you were right - we need `resolveOrgSlug` to first check if `props.entity?.orgSlug1 is defined and return that and if thats not returned return from event.data.

Sorry for the confusion. We do this to ensure that code continues to work the same for people who are already passing `props.entity?.orgSlug` but we dont document it so that new users rely on the automatically inferred org based on event data.